### ISSUE TITLE
[BUG] HTML indent removes code from first line

### ISSUE THREAD

**Issue by melvinmicallef** [yeonjuan/html-eslint#93]
**Bug:**
When using the --fix option for eslint on the rule @html-eslint/indent, if there are multiple html elements on the first line, the code is altered as well. This is very confusing and prone to introducing bugs since elements are removed without notice. Please see replication steps for further info. 

**Replication steps:
Configuration** 
 ```
...
        {
            "files": ["ng-app/**/*.html"],
            "parser": "@html-eslint/parser",
            "rules": {
                "max-lines": ["error", 400],
                "@html-eslint/no-duplicate-id": "error",
                "@html-eslint/no-inline-styles": "error",
                "@html-eslint/no-obsolete-tags": "error",
                "@html-eslint/require-closing-tags": ["error", {"selfClosing": "always"}],
                "@html-eslint/no-duplicate-attrs": "error",
                "@html-eslint/require-img-alt": "error",
                "@html-eslint/no-extra-spacing-attrs": "error",
                "@html-eslint/indent": ["error", 4],
                "@html-eslint/quotes":["error", "double"],
                "@html-eslint/id-naming-convention": ["error", "camelCase"],
                "@html-eslint/no-multiple-empty-lines": "error"
            }
...
```
**Problematic HTML file:**
```
<div class="outer-div"><div class="inner-div">
        inner div
    </div>
</div>
```
**Expected formatting result**
```
<div class="outer-div">
    <div class="inner-div">
        inner div
    </div>
</div>
```

**Actual formatting result - The outer div is removed, breaking the HTML**
```
    <div class="inner-div">
        inner div
    </div>
</div>
```

**Notes:**
This issue only occurs if the two element next to eachother are on the first line of the HTML file. Formatting this file will be done as expected:
```
<div>This is the first line</div>
<div class="outer-div"><div class="inner-div">
        inner div
    </div>
</div>
```
Let me know if I can provide you with further information :)

**Comment by yeonjuan** [comment]
@melvinmicallef  Thanks for the report. I can reproduce it - [demo](https://yeonjuan.github.io/html-eslint/playground#eyJjb2RlIjoiPGRpdiBjbGFzcz1cIm91dGVyLWRpdlwiPjxkaXYgY2xhc3M9XCJpbm5lci1kaXZcIj5cbiAgICAgICAgaW5uZXIgZGl2XG4gICAgPC9kaXY+XG48L2Rpdj4iLCJydWxlcyI6eyJAaHRtbC1lc2xpbnQvaW5kZW50IjoiZXJyb3IiLCJAaHRtbC1lc2xpbnQvcmVxdWlyZS1saS1jb250YWluZXIiOiJlcnJvciJ9fQ==). I'll fix it as soon :)

**Comment by yeonjuan** [comment]
@melvinmicallef Thanks for reminding me. I'm still working on it :) Thanks.

**Comment by yeonjuan** [comment]
Hi @melvinmicallef   It's fixed in v0.14.0 by replacing the parser we use. Thanks for reporting it.
- [demo](https://yeonjuan.github.io/html-eslint/playground#eyJjb2RlIjoiPGRpdiBjbGFzcz1cIm91dGVyLWRpdlwiPjxkaXYgY2xhc3M9XCJpbm5lci1kaXZcIj5cbiAgICAgICAgaW5uZXIgZGl2XG4gICAgPC9kaXY+XG48L2Rpdj4iLCJydWxlcyI6eyJAaHRtbC1lc2xpbnQvaW5kZW50IjoiZXJyb3IiLCJAaHRtbC1lc2xpbnQvcmVxdWlyZS1saS1jb250YWluZXIiOiJlcnJvciIsIkBodG1sLWVzbGludC9lbGVtZW50LW5ld2xpbmUiOiJlcnJvciJ9fQ==)
